scripts/detect-ec2-spot-preemption.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+while true
```

Comment:
Could you document where this script comes from / where the IP address, etc. comes from?